nobl9/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+  "display_name": "Nobl9",
+  "maintainer": "support@nobl9.com",
+  "manifest_version": "1.0.0",
```

Comment:
I believe the `manifest_version` for `extras` integrations should be `2.0.0` now? Is that correct @mgarabed?

Reply:
Nice enthusiasm - but let's hold off on transitioning / migrating other integrations just yet.  We can sync after DASH about what a good approach should be going forward.